scripts/linux/pkg/linux-packaging.md

```diff
@@ -0,0 +1,88 @@
+# RavenDB installation from package on Linux
+
+## What the package does on install
+
+- scatter the files in the default locations
+
+- install systemd service
+
+- add soft links to binaries
+
+- add `ravendb` user and group
+
+- set file permissions properly
+
+- setcap on the Raven.Server
+
+
+## Supported use cases
+
+### Ran under systemd
+The default is when RavenDB is used as a systemd daemon and controlled with systemd.
+The following directory layout (initialized by the package installation process) and permissions is used in this scenario:
+
+#### Filesystem locations and permissions
+
+- settings - `/etc/ravendb/settings.json`
+
+- data - `/var/lib/ravendb/data`
+
+- security (e.g. certificate)  - `/var/lib/ravendb/security`
+
+- logs  - `/var/log/ravendb/logs`
+
+- audit logs  - `/var/log/ravendb/audit`
+
+- binaries (current Server dir) - `/usr/local/ravendb`
+
+- rvn link - `/usr/bin/rvn -> /usr/local/ravendb/rvn`
+
+- ravendb - `/usr/bin/ravendb -> /usr/local/ravendb/Raven.Server` 
+
+- systemd unit file - `/etc/systemd/system/ravendb.service -> /usr/local/ravendb/ravendb.service`
+
+#### Permissions
+
+| Location | Owner | Group | Permissions |
+| - | - | - | - |
+| `/usr/local/ravendb/server` | root | ravendb | 550 |
+| `/usr/bin/rvn -> /usr/local/ravendb/rvn` | root | ravendb | 550 |
+| `/usr/bin/ravendb -> /usr/local/ravendb/Raven.Server` | root | ravendb | 550 |
+| `/etc/ravendb/settings.json` | root | ravendb | 440 |
+| `/var/lib/ravendb/data` | root | ravendb | 770 |
+| `/var/lib/ravendb/security` | root | ravendb | 770 |
+| `/var/log/ravendb/logs` | root | ravendb | 770 |
+| `/var/log/ravendb/audit` | root | ravendb | 770 |
+
+## Running as a regular user
+
+The idea is for the users to be able to run multiple instances of RavenDB on a multi-user system, keeping the data and the settings in their home directory.
+When run as a regular user we start on a random port without settings, producing RavenData dir (and all the other dirs) in the working dir.
```

Comment:
I can't really think of a good scenario for this. If user have installed us, they are likely going to be wanting to use the dameon.

Reply:
Well, I thought about things like a university setting up a multi-user server for all students and then they have the ability to run RavenDB for their purposes. Scrap it?

Reply:
Yes, in almost all such cases, docker containers would be better